src/hb.hh

```diff
@@ -476,6 +476,11 @@ static_assert ((sizeof (hb_var_int_t) == 4), "");
 /* Size signifying variable-sized array */
 #define VAR 1
 
+/* Endian swap, used in Windows related backends */
+static inline uint16_t hb_uint16_swap (const uint16_t v)
+{ return (v >> 8) | (v << 8); }
+static inline uint32_t hb_uint32_swap (const uint32_t v)
+{ return (hb_uint16_swap (v) << 16) | hb_uint16_swap (v >> 16); }
```

Comment:
It is used in all the three hb-directwrite, hb-uniscribe and now hb-gdi thus moved it here above the similar functionality so maybe we merge them later or not.

Reply:
Good.  Though, hb-algs would be better.  Or even better, move near / merge with BEInt stuff in hb-open-type.

hb.hh is typically for macro-intensive platform-specific stuff and trying to minimize it.

Reply:
Ow.  hahahah.  BEInt stuff *is* in hb.hh.  Ignore me.  Thanks.

Reply:
It is above BEInt already :) Agreed that we should consider a merge.